ui/src/app/shared/genericComponents/modal/modal-info-line/modal-info-line.ts

```diff
@@ -8,15 +8,15 @@ import { Icon } from "src/app/shared/type/widget";
 export class ModalInfoLineComponent {
 
     /** Icon, displayed on the left side */
-    @Input() icon: Icon;
+    @Input() protected icon: Icon;
 
     /**
      *  Info-Text, displayed on the right side, optional style for all lines
      *  Multiple lines with own style is possible
      *  */
-    @Input() info: { text: string, lineStyle?: string }[] | string;
+    @Input() public info: { text: string, lineStyle?: string }[] | string;
```

Comment:
protected here as well

Reply:
it is used in  src/app/edge/live/common/selfconsumption/modal/modal.html:2:26 and should be public. it generates error after running build command.